drivers/regulator/regulator_fixed.c

```diff
@@ -86,21 +86,30 @@ static int regulator_fixed_init(const struct device *dev)
 	const struct regulator_fixed_config *cfg = dev->config;
 
 	regulator_common_data_init(dev);
+	bool is_enabled = false;
```

Comment:
Nitpick: Move `is_enabled` two lines up to group it with the other variable declaration